### DISCUSSION TITLE
New suggestion

### DISCUSSION THREAD

**Discussion by TedoJabata** [SSIvanov19/fire-department-2021#59] [Ideas]
Hi,
I really like the project and the idea, and I have suggestion for you! Why don’t you include something like table, from where admins can see other users and employees? And I have one more, why don’t you add options for the admins to edit other employees accounts, teams or cars?

**Comment by SSIvanov19** [comment]
Hi,
Thanks for you suggestion! We are going to look into it. For now you can create an issues and link it here!

**Reply by SSIvanov19** [reply]
Link to the issue #60
